cs/ccxt/ws/OrderBookSide.cs

```diff
@@ -392,7 +409,7 @@ public void storeArray(object delta2)
             var delta = (IList<object>)delta2;
             var price = Convert.ToDecimal(delta[0]);
             var size = Convert.ToDecimal(delta[1]);
-            var order_id = delta[2];
+            var order_id = delta[2] as string;
```

Comment:
we can cast to string here, so no `ToString()` needed down below on multiple places